protocol/x/perpetuals/keeper/perpetual.go

```diff
@@ -738,12 +742,14 @@ func (k Keeper) GetNetNotional(
 	bigNetNotionalQuoteQuantums *big.Int,
 	err error,
 ) {
-	defer telemetry.ModuleMeasureSince(
-		types.ModuleName,
-		time.Now(),
-		metrics.GetNetNotional,
-		metrics.Latency,
-	)
+	if rand.Float64() < 0.01 {
+		defer telemetry.ModuleMeasureSince(
+			types.ModuleName,
+			time.Now(),
+			metrics.GetNetNotional,
+			metrics.Latency,
```

Comment:
Should we add a sampling label with the sampling rate?
(best if using a shared constant that represents the 0.01 and label)